CMSIS/NN/Source/ConvolutionFunctions/arm_convolve_1x1_s8_fast.c

```diff
@@ -72,15 +72,19 @@ arm_status arm_convolve_1x1_s8_fast(const q7_t *input,
                                     const uint16_t output_y,
                                     q15_t *buffer_a)
 {
-    if (input_ch % 4 != 0 ||
-        pad_x != 0 || pad_y != 0 ||
+    if (pad_x != 0 || pad_y != 0 ||
         stride_x != 1 || stride_y != 1)
     {
         return ARM_MATH_SIZE_MISMATCH;
     }
 #if defined(ARM_MATH_MVEI)
     (void)buffer_a;
 
+    if (input_ch % 4 != 0)
```

Comment:
Lets have the same condition for MVE and DSP. I'll add a ticket to remove this constraint on MVE..